### PR TITLE
Witnet 1.5

### LINKED ISSUE #2129: Implement StringtoXml radon operator
Implement radon operator `String.toXml` which parses a string containing an XML to a Map

### LINKED ISSUE #2130: Add HTTP-POST as witnet capability
Currently, the Currently, witnet-rust only supports HTTP-GET as a source type. To allow data request creators to choose more sources, we have to add support for HTTP-POST

### PR DESCRIPTION
**Pull request by tmpolaczyk**
I'm opening this draft to facilitate review and testing of the upcoming 1.5 release.

The release date has not been decided yet, and we may want to add some other features.

Close #2129
Close #2130

### CODE REVIEW

File: data_structures/src/mainnet_validations.rs [hunk 1]
```diff
@@ -119,21 +131,51 @@ impl TapiEngine {
...
+        let mut voting_wips = vec![None; 32];
 
         match environment {
             Environment::Mainnet => {
                 // Hardcoded information about WIPs
                 for (k, v) in wip_info() {
                     self.wip_activation.insert(k, v);
                 }
+
+                // TODO: Use right WIP number and properties
```
Comment (tmpolaczyk) [inline data_structures/src/mainnet_validations.rs:143]:
```suggestion
```

This TODO is now fixed, right?

File: rad/src/lib.rs [hunk 2]
```diff
@@ -266,25 +291,93 @@ async fn rng_response(
...
+}
+
 /// Run retrieval stage of a data request, return `Result<RadonReport>`.
 pub async fn run_retrieval_report(
     retrieve: &RADRetrieve,
     settings: RadonScriptExecutionSettings,
     active_wips: ActiveWips,
 ) -> Result<RadonReport<RadonTypes>> {
     let wip_0019_active = active_wips.wip0019();
+    let wip0020_active = active_wips.wip0020();
```
Comment (tmpolaczyk) [inline rad/src/lib.rs:367]:
```suggestion
    let wip_0019_active = active_wips.wip0019();
    let wip_0020_active = active_wips.wip0020();
```

**Comment by tmpolaczyk** [comment]
Approve

**Comment by tmpolaczyk** [comment]
Approve

**Review by lrubiorod — APPROVED** [review]

**Pull request merged**